### ISSUE TITLE
Prioritising workshop material suitable for non-native English speakers

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/master-reference#890]
On today's CF call, concerns were expressed about the wordiness of some of the workshop material. @MynahMarie, @ibraghada  and @yasminhillis brought up the question of localising some of the material. @lucymk expressed  a preference for simplifying the language on master reference first. All agreed that they see simplifying the language in workshops as a priority for any new material created on master reference.

**Comment by eliascodes** [comment]
Agree in general

**Comment by astroash** [comment]
I put this document together whilst I was CF, which listed things I thought were important to keep language easy to understand: [how to write documentation](https://hackmd.io/Q7O6DwpmSCCbH2lWqxkaWA#). 

I was wondering if it would be useful to draw on alumni support in getting some resources rewritten, we could ask people to pick one workshop to test and reword against the above criteria. I've often found that peeps like to help out with FAC things, but it's often difficult to find low commitment tasks to help out with. Happy to spearhead this if people think it's a good idea.

**Comment by MynahMarie** [comment]
@astroash Yes definitely we could get alumni support for tasks like rewording/rewriting 😸  but I think that maybe first we need to know how exactly we will organize the documentation? Are we making different branches for different campuses? or are we going to have have different references for each campus? Honestly I'm not sure what would be the best way to approach this if we try to think ahead (when FAC will have dozens of campuses around the World 😺 ) and find a solution that would be scalable in the long run... What does everyone think?

**Comment by eliascodes** [comment]
To clarify, I was agreeing with this sentiment:
> expressed a preference for simplifying the language on master reference first

I'm assuming by "master reference", @lucymk means the curriculum as a whole, not just this repo. Anyway, this is something that can be done with little to no delay or planning. 

Questions around localisation, and those @MynahMarie raised around organisation of resources for different campuses are much bigger and require more discussions and thought. We've thought about forking MR and variations on that theme for a while without coming to a satisfactory conclusion.

**Comment by m4v15** [comment]
>Are we making different branches for different campuses? or are we going to have have different references for each campus? Honestly I'm not sure what would be the best way to approach this if we try to think ahead (when FAC will have dozens ssof campuses around the World 😺 ) 

This is definitely something we need to think about strategically going forward but not sure it needs to be solved before doing smaller things that might improve current workshops.

like @eliascodes I agree in general, but I'm wary on a few things:
   - Different people have different ideas of what a good rewording might be - things could get reworded now, and then get reworded back to something similar to how they were before depending on what another mentor see's as appropriate
   - If we make the WS's less wordy does that mean they will require more hands on mentor notes for delivery?

However, this are just slight concerns, @astroash I think you should go ahead and try and spearhead some people making some changes to some workshops. If they make PR's we can have discussions about the above 2 points in those PR's. I think do it in whatever way you see fit, start on a few WS's and see how it goes.

**Comment by shiryz** [comment]
@m4v15, I think having a clear process on how to simplify would be preferable, the MR has an inconsistent format as it is... it doesn't have to be a rigid process it can be a set of rules for people to follow :man_dancing: 

In general, I do agree that some WSs are quite wordy and hard for people to follow, I would love to help change some :smile_cat:

**Comment by astroash** [comment]
@MynahMarie , @shiryz, @yasminhillis, @ibraghada if we were to have tips on how to simplify language for non native english speakers would you add anything else to the list on [this document](https://hackmd.io/Q7O6DwpmSCCbH2lWqxkaWA#)? _The section on planning and structure can be removed if we're handing out as a rewrite guide._

Also is there any particular workshops that have been highlighted by students as difficult to understand?

**Comment by m4v15** [comment]
Yeah, sorry @shiryz I guess I meant part of spearheading it would be coming up with a set of guidelines, and then a set of workshops that most need updating, and then getting some PR's.

Regarding these guidelines, writing in english for non-native audiences is definitely a common problem that others have thought about a lot, a quick google threw up up few things that might be helpful - to save a bit of mental energy on our part, it makes sense to use something like this to help inform our own guidelines (I'm sure you probably did this already @astroash !):
- https://authorunlimited.com/writing-for-non-native-english-speakers/
- http://writing.msu.edu/writing-for-non-native-speakers-of-english/
- https://uxmag.com/articles/writing-english-for-global-audiences
- https://www.apc.org/sites/default/files/Writing_in_English_for_a_multilingual_audience.pdf

**Comment by haydnba** [comment]
I would really like to help out on this - happy to support core campus assignees. I definitely agree the master-reference should be handled first before perhaps looking at dedicated localised branches. One key issue to mention is that the master-reference should be __de-localised__ as much as possible - that is, change 'Monday, Tuesday, Weds...' to 'Day 1, Day 2, Day 3....' and remove suggestions that students might want to build an app that uses the TFL api to check local bus services to the Finsbury Park campus. This kind of stuff really makes the students here in Nazareth feel a bit sidelined...

**Comment by EarthToAbigail** [comment]
@astroash I think it would be good to have non-native english speakers working on the re-wording of workshops... I think it might be difficult for people who have english as a native language to efficiently spot the words or phrases that might be difficult to understand. I will go over the list of workshops we've done so far and the SGC notes of the past weeks, will comment on this issue with a list of the workshops we adapted for non-native english speakers so far (might take me a few days so please be patient with me... ).

**Comment by MynahMarie** [comment]
@astroash I think it would be good to have non-native english speakers working on the re-wording of workshops... I think it might be difficult for people who have english as a native language to efficiently spot the words or phrases that might be difficult to understand. I will go over the list of workshops we've done so far and the SGC notes of the past weeks, will comment on this issue with a list of the workshops we adapted for non-native english speakers so far (might take me a few days so please be patient with me... ).
